walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,120 @@
 
 package com.google.sps;
 
-import java.util.Collection;
+import java.util.*;
 
+/*
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
     throw new UnsupportedOperationException("TODO: Implement this method.");
   }
-}
+}*/
+
+
+public final class FindMeetingQuery {
+  public boolean hasMatch(HashSet<String> first, HashSet<String> second) {
```

Comment:
first and second as argument names sound unclear as to what they hold

Reply:
Okay, what should I change it to? I saw on `Event.java` file that they used `a` and `b` for this kind of compare function. Should I change it to `a` and `b`?